src/zimscraperlib/image/conversion.py

```diff
@@ -40,6 +42,39 @@ def convert_image(
         save_image(image, dst, fmt, **params)
 
 
+def convert_svg2png(
+    src: str | pathlib.Path | io.BytesIO,
+    dst: pathlib.Path | IO[bytes],
+    width: int | None = None,
+    height: int | None = None,
+):
+    """Convert a SVG to a PNG
+
+    Output width and height might be specified if resize is needed.
+    PNG background is transparent.
+    """
+    kwargs = {}
+    if isinstance(src, pathlib.Path):
+        src = str(src)
+    if isinstance(src, str):
+        kwargs["url"] = src
+    else:
+        kwargs["bytestring"] = src.getvalue()
+    if width:
+        kwargs["output_width"] = width
+    if height:
+        kwargs["output_height"] = height
+    if isinstance(dst, pathlib.Path):
+        cairosvg.svg2png(write_to=str(dst), **kwargs)
+    else:
+        result = cairosvg.svg2png(**kwargs)
+        if not isinstance(result, bytes):
```

Comment:
What's the reason for this? I see it's not covered so I guess you can't trigger it

Reply:
For some reason the type hint of `cairosvg.svg2png` return value is `bytes | Unknown | None`. I have no idea why, but I prefer to raise a clear exception should this happen so that it will be easier to debug. Probably only a typing issue, but still better safe than sorry.

Reply:
For some reason, `cairosvg.svg2png` return type is detected as `bytes | Unknown | None`. I prefer to log proper exception should this happen, even if I have no idea how to trigger this (yet). The proper exception will help to diagnose the issue far more than an obscure message linked to the fact that improper type has been returned and does not has proper member